clinica/iotools/converters/specifications/sessions.tsv

```diff
@@ -1,431 +1,431 @@
+	clinad_volbrain_lateral_ventricles_right_cm3		float			Lateral ventricles Right cm3
+	clinad_volbrain_lateral_ventricles_right_%		float			Lateral ventricles Right %
+	clinad_volbrain_lateral_ventricles_left_cm3		float			Lateral ventricles Left cm3
+	clinad_volbrain_lateral_ventricles_left_%		float			Lateral ventricles Left %
+	clinad_volbrain_lateral_ventricles_asymmetry		float			Lateral ventricles Asymmetry
+	clinad_volbrain_caudate_total_cm3		float			Caudate Total cm3
+	clinad_volbrain_caudate_total_%		float			Caudate Total %
+	clinad_volbrain_caudate_right_cm3		float			Caudate Right cm3
+	clinad_volbrain_caudate_right_%		float			Caudate Right %
+	clinad_volbrain_caudate_left_cm3		float			Caudate Left cm3
+	clinad_volbrain_caudate_left_%		float			Caudate Left %
+	clinad_volbrain_caudate_asymmetry		float			Caudate Asymmetry
+	clinad_volbrain_putamen_total_cm3		float			Putamen Total cm3
+	clinad_volbrain_putamen_total_%		float			Putamen Total %
+	clinad_volbrain_putamen_right_cm3		float			Putamen Right cm3
+	clinad_volbrain_putamen_right_%		float			Putamen Right %
+	clinad_volbrain_putamen_left_cm3		float			Putamen Left cm3
+	clinad_volbrain_putamen_left_%		float			Putamen Left %
+	clinad_volbrain_putamen_asymmetry		float			Putamen Asymmetry
+	clinad_volbrain_thalamus_total_cm3		float			Thalamus Total cm3
+	clinad_volbrain_thalamus_total_%		float			Thalamus Total %
+	clinad_volbrain_thalamus_right_cm3		float			Thalamus Right cm3
+	clinad_volbrain_thalamus_right_%		float			Thalamus Right %
+	clinad_volbrain_thalamus_left_cm3		float			Thalamus Left cm3
+	clinad_volbrain_thalamus_left_%		float			Thalamus Left %
+	clinad_volbrain_thalamus_asymmetry		float			Thalamus Asymmetry
+	clinad_volbrain_globus_pallidus_total_cm3		float			Globus Pallidus Total cm3
+	clinad_volbrain_globus_pallidus_total_%		float			Globus Pallidus Total %
+	clinad_volbrain_globus_pallidus_right_cm3		float			Globus Pallidus Right cm3
+	clinad_volbrain_globus_pallidus_right_%		float			Globus Pallidus Right %
+	clinad_volbrain_globus_pallidus_left_cm3		float			Globus Pallidus Left cm3
+	clinad_volbrain_globus_pallidus_left_%		float			Globus Pallidus Left %
+	clinad_volbrain_globus_pallidus_asymmetry		float			Globus Pallidus Asymmetry
+	clinad_volbrain_hippocampus_total_cm3		float			Hippocampus Total cm3
+	clinad_volbrain_hippocampus_total_%		float			Hippocampus Total %
+	clinad_volbrain_hippocampus_right_cm3		float			Hippocampus Right cm3
+	clinad_volbrain_hippocampus_right_%		float			Hippocampus Right %
+	clinad_volbrain_hippocampus_left_cm3		float			Hippocampus Left cm3
+	clinad_volbrain_hippocampus_left_%		float			Hippocampus Left %
+	clinad_volbrain_hippocampus_asymmetry		float			Hippocampus Asymmetry
+	clinad_volbrain_amygdala_total_cm3		float			Amygdala Total cm3
+	clinad_volbrain_amygdala_total_%		float			Amygdala Total %
+	clinad_volbrain_amygdala_right_cm3		float			Amygdala Right cm3
+	clinad_volbrain_amygdala_right_%		float			Amygdala Right %
```

Comment:
Could you explain what was done to these files ? The diff super hard to make sense of...

Reply:
These files hardcode where specific metadata keys are stored. For OASIS, everything was inside 'oasis_cross-sectional.csv'. Every occurence I changed it to 'oasis_cross-sectional-5708aa0a98d82080.xlsx'

Reply:
Thanks ! I think I got confused because there are a lot of lines which were ending with either tabs or spaces which were removed, resulting in this huge diff.